pkg/kubernetes/client/client.go

```diff
@@ -51,6 +51,9 @@ type ApplyOpts struct {
 	// force allows to ignore checks and force the operation
 	Force bool
 
+	// disabled validate allows to skip validation
```

Comment:
```suggestion
	// Validate allows to enable/disable kubectl validation
```

Reply:
applied this one and previous one (just changed to lower case, as in other options)